### PR TITLE
TILA-1394 | static files serving

### PR DESCRIPTION
**Pull request by Niilis**
Application started dropping connections when the application was under load. These changes scale-up amount of workers when load increases. Static files are also served in an unconventional way. Documentation is updated to ease understanding of potential failures in the future.

* Improve application performance on-load.
* Improve documentation

Refs TILA-1394

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,3 +127,11 @@ For development purposes requirement to authenticate can be turned off by settin
 environment variable TMP_PERMISSIONS_DISABLED to True.
 
 ![Data model visualization](tilavarauspalvelu_visualized.svg)
+
+## Deployed application settings & static files
+
+Contarary to more common set-ups, this application does not have a reverse proxy serving static files. Instead, static files are served by both Django and wsgi.
```
Comment (nicobav) [inline README.md:133]: Typo in `Contrary`

**Review by nicobav — CHANGES_REQUESTED** [review]
Looks good :+1: 

Very tiny miney minor typo in the README section.

File: README.md (hunk 1 above)
Comment (Niilis) [reply README.md]: Oh no! I should start reading what I write more often :sweat_smile:

**Review by nicobav — APPROVED** [review]
Yay! :nerd_face:

**Pull request merged**